### PR TITLE
AEA-313 First draft for quickstart

### LINKED ISSUE #766: Setup instructions additions
**Is your feature request related to a problem? Please describe.**
Currently, the setup instructions do not cover a special case when the user uses `zsh` rather than `bash` in terminal.

**Describe the solution you'd like**
Explain, that when the user uses `zsh` they need  to do `pip install 'aea[all]'`

### PR DESCRIPTION
**Pull request by Totoual**
## Proposed changes

Changes the Quickstart guide to follow https://docs.fetch.ai/getting-started/quickstart/

## Fixes
  
#766 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Totoual** [comment]
Please give me feedback on how we want the document. I just tried to make the Quickstart section smaller and added some links.

File: docs/quickstart.md [hunk 1]
```diff
@@ -68,52 +67,9 @@ sudo apt-get install python3.7-dev
 
 ## Echo skill demo
 
-The echo skill is a simple demo that introduces you to the main business logic components of an AEA.
-
-## Option 1 - Step-by-step
```
Comment (DavidMinarsch) [inline docs/quickstart.md]: Can we move this option to the bottom and make it `toggle-able`?

File: docs/quickstart.md [hunk 2]
```diff
@@ -181,5 +144,55 @@ Delete the AEA from the parent directory (`cd ..` to go to the parent directory)
...
+<br>  
+<b>Add the echo skill</b> 		
+<br>    
+Second, add the echo skill to the project.		
+```bash
+aea add skill fetchai/echo:0.1.0		
+```		
+This copies the `echo` skill code containing the "behaviours", "handlers", and "tasks" into the skill, ready to run. The identifier of the skill `fetchai/echo:0.1.0` consists of the name of the author of the skill, followed by the skill name and its version.		
+<br><br>
+<b>Add a stub connection</b>
```
Comment (DavidMinarsch) [inline docs/quickstart.md]: The section on the stub connection must be in the main bit, so lines 177 - 198

File: docs/quickstart.md [hunk 3]
```diff
@@ -82,6 +82,28 @@ aea fetch fetchai/my_first_aea:0.1.0
 cd my_first_aea
 ```
 
+## Add a stub connection
```
Comment (DavidMinarsch) [inline docs/quickstart.md]: Change to `Usage of the stub connection`

**Pull request merged**